homeworks/hw2/stackCalculator/StackOnLinkedList.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace StackCalculator;
+
+using System.Collections;
+using System.Collections.Generic;
+
+// Stack implementation using a linked list
+internal class StackOnLinkedList : IStack
+{
+
+    public StackOnLinkedList()
+    {
+        this.data = new LinkedList<double>();
+    }
```

Comment:
```suggestion
    public StackOnLinkedList()
        => this.data = new();
```
Тип this.data известен из его объявления, поэтому компилятор тут разберётся.